cmd/snap/cmd_help_test.go

```diff
@@ -97,7 +98,26 @@ func nonHiddenCommands() map[string]bool {
 	return names
 }
 
+// Helper that checks if goflags is old. The check for EnvNamespace is
+// arbitrary, it just happened that support for this got added right after
+// the v1.4.0 release with commit 1c38ed7.
```

Comment:
This is fine, and the test will execute in the development and Ci environments as the commit we vendor (7309ec74f752d05ce2c62b8fd5755c4a2e3913cb)  is 34 revisions later.